### ISSUE TITLE
settings not setting

### ISSUE THREAD

**Issue by lerker100** [artemanufrij/webpin#85]
In the settings menu if I click the Save login information button and save the web app everything looks fine.  Then I close the webpin settings menu and reopen it reverts back to unchecked and the same thing happens for the save cookies button.

**Comment by NathanBnm** [comment]
Yes, and when you want to make some change on an app, the "Save" button can't be pressed, so you can't modify any app, you're forced to delete and recreate it.

**Comment by lerker100** [comment]
Yes

**Comment by TeNNoX** [comment]
When I click 'Edit' on an existing app - I see this screen:  
![Screenshot from 2019-11-24 15-26-06](https://user-images.githubusercontent.com/2084639/69496873-cd619e00-0ece-11ea-9023-b3c2c5906f95.png)

Note that it says "Create new web app", and the Save button is greyed out. No ability to save changes

**Comment by TeNNoX** [comment]
I figured out that this is caused by the icon being invalid (which is visually indicated but not obvious why this would cause the Save to be disabled).

I tackled this bug which causes the invalid icon and also improved the UX of invalid fields in this PR:  
https://github.com/artemanufrij/webpin/pull/121
